themes/src/main/resources/theme/keycloak/welcome/index.ftl

```diff
@@ -4,7 +4,27 @@
     <meta charset="utf-8">
     <meta name="robots" content="noindex, nofollow">
     <meta name="viewport" content="width=device-width, initial-scale=1">
+    <meta name="color-scheme" content="light dark">
     <title>Welcome to ${productName}</title>
+    <script type="module" async blocking="render">
```

Comment:
Same here, this should be conditional based on whether the dark mode is enabled.